Changelog.md

```diff
@@ -1,5 +1,30 @@
 ## Gazebo Math 7.x
 
+### Gazebo Math 7.2.0 (2023-08-29)
```

Comment:
7.3.0

Reply:
Thanks for catching that. https://github.com/gazebosim/gz-math/pull/552/commits/1237176f39955e10e3a8fb2e7e3c50ede7b66baa